api/src/com/cloud/storage/Volume.java

```diff
@@ -52,7 +52,8 @@
         UploadInProgress("Volume upload is in progress"),
         UploadError("Volume upload encountered some error"),
         UploadAbandoned("Volume upload is abandoned since the upload was never initiated within a specificed time"),
-        Attaching("The volume is attaching to a VM");
+        AttachingFromReady("The volume is attaching to a VM from Ready State"),
```

Comment:
@anuragaw don't change the original state `Attaching` to honour db volume states that may already be in `Attaching` state in db. This may cause usage issues post an upgrade.

Reply:
Ah, my bad. Fixing.